pkgs/tools/admin/scaleway-cli/default.nix

```diff
@@ -1,18 +1,19 @@
-{ lib, fetchFromGitHub, buildGoPackage }:
+{ lib, fetchFromGitHub, buildGoModule }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "scaleway-cli";
-  version = "1.20";
-
-  goPackagePath = "github.com/scaleway/scaleway-cli";
+  version = "2.4.0";
 
   src = fetchFromGitHub {
     owner = "scaleway";
     repo = "scaleway-cli";
     rev = "v${version}";
-    sha256 = "14likzp3hl04nq9nmpmh9m5zqjyspy5cyk20dkh03c1nhkd4vcnx";
+    sha256 = "yYzcziEKPSiMvw9LWd60MkHmYFAvN7Qza6Z117NOOv0=";
   };
 
+  vendorSha256 = "0V9sHi/E095txnfF8YFW5O7o0e1H3sdn3tw5LqB92tI=";
+  doCheck = false;
```

Comment:
Please leave a comment why you've disabled the tests. Are they broken? Are there none? That sort of thing.

Reply:
Some of the tests require network access to scaleway's API, and therefore break when sandboxed. Ideally those would be selectively disabled, but I'm not familiar enough with go's tooling to do this.